.gitignore

```diff
@@ -12,12 +12,11 @@
 
 # Package Files #
 *.jar
-*.war
-*.nar
-*.ear
 *.zip
-*.tar.gz
-*.rar
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+.idea/.gitignore
+.idea/compiler.xml
+.idea/misc.xml
+.idea/sonarlint/
```

Comment:
https://github.com/companieshouse/third-party-test-harness/pull/1#discussion_r522840240

Reply:
Don't know how these crept back in will remove again!